### PR TITLE
[android] update nav icons to match previous design

### PR DESCRIPTION
**Pull request by arnaudvergnet**
First step of https://github.com/organicmaps/organicmaps/issues/3406.

Took the icons from [Material Design Icons](https://materialdesignicons.com/) and manually edited the _follow and rotate_ to add the line.

| not follow | pending | follow | follow and rotate |
|-|-|-|-|
| Screenshot_1663161694 | Screenshot_1663161698 | Screenshot_1663161711 | Screenshot_1663161713 |
| | | navigation-variant | navigation |

### CODE REVIEW

**Review by biodranik — CHANGES_REQUESTED** [review]
Why do you want to change the existing UX (follow mode = _filled_ blue arrow)?

Would it be better to use your current _not filled_ blue icon instead of the crosshair, when _there is_ a position, but it's not followed? And use a filled (rotated) arrow (as before) for the follow mode?
And of course, we need to introduce a new icon, when the position is not known at all, and is not actively searched (when it's searched there should be a radar-like animation, right?)

Also, It would be better to increase the size of the arrows inside buttons (without increasing buttons size!).

**Comment by arnaudvergnet** [comment]
Oh yes sorry I forgot the base icon was filled I will change that. I won't change the UX in this PR, it was only meant to add the previous icons but in SVG format.

**Comment by arnaudvergnet** [comment]
Updated the main post and code to use a filled icon in follow mode.

**Comment by biodranik** [comment]
Thanks! Aren't they smaller than before, and smaller than zoom buttons? Will the same zoom buttons size trick work?

**Comment by arnaudvergnet** [comment]
Yes the arrow icons appear smaller.

In the previous implementation, the padding was applied inside the icon image. Now it is applied in the button. This is a lot better as it allows reusing the same icons for other purposes.

So previously the size was handled while designing the icon. Now the padding is handled by the button itself so I need to specify the size in the styles (this is what I did for the zoom). 

In this case, the "not follow icon" is at the right size, so I need to dynamically change the button padding when showing the arrows for them to appear bigger (and not change the size of the "not follow icon").


I have a fix working I just need to tweak the settings to get an appropriate size.

**Comment by arnaudvergnet** [comment]
@biodranik I updated the main post with the new screenshots.

File: android/src/com/mapswithme/maps/widget/menu/MyPositionButton.java [hunk 1]
```diff
@@ -47,9 +48,17 @@ public void update(int mode)
...
+    @AttrRes int colorAttr = R.attr.iconTint;
+    @DimenRes int sizeDimen = R.dimen.map_button_icon_size;
+    if (mode == LocationState.FOLLOW || mode == LocationState.FOLLOW_AND_ROTATE || mode == LocationState.PENDING_POSITION)
+    {
+      colorAttr = R.attr.colorAccent;
+      if (mode == LocationState.PENDING_POSITION)
+        sizeDimen = R.dimen.map_button_size;
+      else
+      sizeDimen = R.dimen.map_button_arrow_icon_size;
+
```
Comment (vng) [inline android/src/com/mapswithme/maps/widget/menu/MyPositionButton.java:60]: indent and remove empty string

Comment (arnaudvergnet) [reply android/src/com/mapswithme/maps/widget/menu/MyPositionButton.java:60]: Fixed the indent but what do you mean by empty string?

**Review by biodranik — APPROVED** [review]
A proper (simpler) solution would be to use icons of a similar size (or resize them) and avoid dynamic size switching in the code. But for now, let's proceed with the beta/release.

File: android/src/com/mapswithme/maps/widget/menu/MyPositionButton.java (hunk 1 above)
Comment (biodranik) [reply android/src/com/mapswithme/maps/widget/menu/MyPositionButton.java]: Empty line?

**Pull request merged**